src/compiler/emitter.ts

```diff
@@ -42,6 +42,14 @@ var __assign = (this && this.__assign) || Object.assign || function(t) {
     return t;
 };`;
 
+        const restHelper = `
+var __rest = (this && this.__rest) || function (s, e) {

```

Comment:
also you will need a check in checker.ts for __rest and __assign, and add some tests for these.